.evergreen/config_generator/components/funcs/upload_test_results.py

```diff
@@ -10,7 +10,7 @@ class UploadTestResults(Function):
         # Ensure attach_results does not fail even if no tests results exist.
         bash_exec(
             script='''\
-                mkdir mongoc
+                mkdir -p mongoc
```

Comment:
Neat.  TIL that one of the uses of the `-p` to `mkdir` is "no error if existing".